CONTRIBUTING.md

```diff
@@ -37,6 +37,47 @@ Note that we welcome all contributions, e.g., further protocol models. Just
 send us a pull request.
 
 
+Making new releases
+-------------------
+
+The Tamarin Prover is distributed through a number of channels: source and binaries through Github, and packages through Homebrew, Arch and NixOS. When a new release is cut, here's how to update the various distribution channels.
+
+1. Make the new release on Github. @rsasse or @jdreier usually does this.
+
+   1. run 'make case-studies' and compare 'case-studies/' with
+      'case-studies-regression/'
+
+   2. run 'make sapic-case-studies' and compare 'case-studies-sapic/' with
+      'case-studies-sapic-regression/'
+
+   3. run 'tamarin-prover test'
+
+   4. call 'tamarin-prover' and copy CSF'12 automatic command and
+      check whether verification succeeds.
+
+   5. generate 'intruder_variants_{dh,bp}.spthy' and diff with versions in data/
```

Comment:
how do you generate these?

Reply:
Explained and added the resulting output file (as both get created in one shot).